Jenkinsfile

```diff
@@ -0,0 +1,97 @@
+G_gitcred = 'TonJenSSH'
+G_container = "atomxy/empty-ton-sdk-js:20191128"
+G_update = "none"
+G_build = "none"
+G_test = "none"
+
+pipeline {
+    triggers {
+        upstream(
+            upstreamProjects: 'Node/ton-labs-types/master',
+            threshold: hudson.model.Result.SUCCESS
+        )
+    }
+    options {
+        buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '1')
```

Comment:
`numToKeepStr` is set to `1`. Could we keep more than 1 last build? I would propose to use default value, `10`

Reply:
Do you really think that we need to keep builds reports for subproject? I think it might be sensible for main project. This pipeline was been added just for correct triggering other pipelines